### ISSUE TITLE
Request: Support for Descript Audio Codec

### ISSUE THREAD

**Issue by GUUser91** [Plachtaa/VALL-E-X#130]
With Descript Audio Codec, you can compress 44.1 KHz audio into discrete codes at a low 8 kbps bitrate. This universal model works on all domains (speech, environment, music, etc.), making it widely applicable to generative modeling of all audio. It can be used as a drop-in replacement for EnCodec for all audio language modeling applications (such as AudioLMs, MusicLMs, MusicGen, etc.)
https://github.com/descriptinc/descript-audio-codec

**Comment by Plachtaa** [comment]
I agree that descript audio codec is a better option, but you have to retrain the whole model
